Makefile

```diff
@@ -1,28 +1,36 @@
-ASSETS := $(shell yq e '.assets.[].src' manifest.yaml)
-VERSION := $(shell yq e ".version" manifest.yaml)
-S9PK_PATH=$(shell find . -name gitea.s9pk -print)
+PKG_VERSION := $(shell yq e ".version" manifest.yaml)
+PKG_ID := $(shell yq e ".id" manifest.yaml)
 TS_FILES := $(shell find . -name \*.ts )
 
 # delete the target of a rule if it has changed and its recipe exits with a nonzero exit status
 .DELETE_ON_ERROR:
 
 all: verify
 
-install: gitea.s9pk
-	embassy-cli package install gitea.s9pk
+# assumes /etc/embassy/config.yaml exists on local system with `host: "http://embassy-server-name.local"` configured
+install: $(PKG_ID).s9pk
+	embassy-cli package install $(PKG_ID).s9pk
 
-verify: gitea.s9pk $(S9PK_PATH)
-	embassy-sdk verify s9pk $(S9PK_PATH)
+verify: $(PKG_ID).s9pk
+	embassy-sdk verify s9pk $(PKG_ID).s9pk
 
 clean:
+	rm -rf docker-images
+	rm -f  $(PKG_ID).s9pk
 	rm -f image.tar
-	rm -f gitea.s9pk
+	rm -f scripts/*.js
 
-gitea.s9pk: manifest.yaml image.tar instructions.md scripts/embassy.js
+ $(PKG_ID).s9pk: manifest.yaml instructions.md scripts/embassy.js gitea/LICENSE docker-images/aarch64.tar docker-images/x86_64.tar
```

Comment:
There's a space at the beginning of this line, which causes Make to puke

Reply:
fixed